charters/ExpressionMatrixService/charter.md

```diff
@@ -0,0 +1,93 @@
+
+# Expression Matrix Service
+
+## Description
+
+The expression matrix service enables researchers to easily access and analyze HCA expression matrix data.
+
+## Objective
+
+* Build, operate, and maintain a data access API that serves expression
+matrices from the HCA DCP Data Store with options to subset data based on bundle metadata, genes, and numerical filters.
+
+* Serve as a test bed for a re-usable metadata query service, created by
+the *Query Service* component.
+
+## In-scope
+
+* Expression Matrix Service interface supports queries based on:
+    * Bundle identifiers
+        * This query enables selection of cells from specific submissions. 
+    * Metadata (*across bundles; potentially provided directly by Query Module*)
```

Comment:
So just to be clear, the matrix service will index metadata in bundles itself (or use another index like ES or Postgres) to understand metadata queries?  Just asking because on the orange side, we're planning on just handing off a list of bundle IDs/versions but if the matrix service "understands" the HCA metadata then more interesting query handoff may be possible.  On the orange side we'd need to understand more about the query abilities for the matrix service but for now I think the handoff of bundle UUIDs and versions is sufficient.